src/cultionet/model.py

```diff
@@ -328,27 +376,29 @@ def fit(
 
     # Checkpoint
     cb_train_loss = ModelCheckpoint(
+        monitor='loss'
+    )
+    # Validation and test loss
+    cb_val_loss = ModelCheckpoint(
         dirpath=ckpt_file.parent,
         filename=ckpt_file.stem,
         save_last=True,
         save_top_k=save_top_k,
         mode='min',
-        monitor='loss',
+        monitor='val_score',
         every_n_train_steps=0,
         every_n_epochs=1
     )
-    # Validation and test loss
-    cb_val_loss = ModelCheckpoint(monitor='val_loss')
     # Early stopping
     early_stop_callback = EarlyStopping(
-        monitor='val_loss',
+        monitor='val_score',
```

Comment:
We are now monitoring a combined validation score instead of the validation loss.

Reply:
nice 👍